### DISCUSSION TITLE
How to use the boilerplate without database?

### DISCUSSION THREAD

**Discussion by Tasin5541** [hagopj13/node-express-boilerplate#201] [Q&A]
This might be a silly question, but how do I remove all mongoDB related files and dependencies (dockerfile too)?
I like the boilerplate structure and would like to use it. But I don't need mongoDB or any other DB at the moment as I will be using this api to call another api that'll handle the db related works.

**Comment by jleei** [comment]
There is a lot of work to be done, removing all MongoDB related stuff, calling external apis in the service layer.
